experiments/mpi-pingpong/experiment.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchpark.directives import variant, maintainers
+from benchpark.experiment import Experiment
+from benchpark.caliper import Caliper
+from benchpark.affinity import Affinity
+
+
+class MpiPingpong(Experiment, Caliper, Affinity):
+
+    variant(
+        "workload",
+        default="run",
+    )
+
+    maintainers("stephanielam3211")
+
+    def compute_applications_section(self):
+        n_ranks = 448   #dane:896, ruby:448, tioga:512, lassen:352, tuolumne:768
```

Comment:
We need to set this in terms of nodes `8`, so we don't have to specify this for each machine. However the affinity output will look different, so we need to figure this out.